src/main/java/edu/illinois/cs/dt/tools/diagnosis/TestDiagnoser.java

```diff
@@ -36,7 +36,12 @@ public TestDiagnoser(final InstrumentingSmartRunner r, final MinimizeTestsResult
         System.out.println();
         System.out.println("-----------------------------------------------------------------");
         System.out.println("Running diagnoser for " + minimized.dependentTest() + " (expected result in this order: " + minimized.expected() + ")");
-        System.out.println(minimized.deps().size() + " known dependencies: " + minimized.deps());
+        if (minimized.polluters().size() > 0) { // Did it find any polluters? Print for first one
+            System.out.println(minimized.polluters().get(0).deps().size() + " known dependencies: " + minimized.polluters().get(0).deps());
+        }
+        else {
```

Comment:
Very minor, but I always put the else on the same line as the end bracket for the if (line 41 in this case), so I think it would be best to keep it consistent.